warehouse/models/mart/payments/_payments.yml

```diff
@@ -269,7 +276,11 @@ models:
         description: '{{ doc("lp_settlement_id") }}'
       - *organization_name
       - *organization_source_record_id
-      - name: end_of_month_date
+      - name: end_of_month_date_pacific
+        description: |
+          The last day of the month in Pacific Time (involves conversion of record_updated_timestamp_utc to Pacific Time).
+          This column is primarily for use in BI tooling to support consistent dates across activity types.
+      - name: end_of_month_date_utc
         description: |
           The last day of the month of the record_updated_timestamp_utc.
           This column is primarily for use in BI tooling to support consistent dates across activity types.
```

Comment:
Should this description line for `end_of_month_date_utc` be modified to reflect updated patterns of use?

Reply:
pushed update:

> This column preserves the end of month date value for the original UTC date as received by Littlepay, however, `end_of_month_date_pacific` is primarily used in BI tooling instead to support consistent dates across activity types.